packages/atlas/lib/src/circle.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:flutter/foundation.dart';
+import 'package:atlas/atlas.dart';
+import 'package:flutter/material.dart';
+
+/// `Circle` with specified center/radius on the map
+class Circle {
+  /// Uniquely identifies a `Circle`.
+  final String id;
+
+  /// The location where the `Circle` is centered. Is represented as `LatLng`.
+  final LatLng center;
+
+  /// The radius of the `Circle`.
+  final double radius;
```

Comment:
What's the unit of the `radius`? I think a `double` type might not be enough, or we should change then the name of the variable...

Reply:
For both [Google Maps API](https://developers.google.com/android/reference/com/google/android/gms/maps/model/Circle.html#getRadius()) and [HERE Maps API](https://developer.here.com/documentation/maps/topics_api/h-map-circle.html#h-map-circle__getradius) it seems to be a double-precision 64-bit floating point..

Reply:
I understand that, but here's what I mean; you could have this code:

```dart
final circle = Circle(
...
radius: 23.9087,
...
);
```

Is that radius miles, kilometers...? What if a different map provider, like mapbox, had a different API for this? That's why I think it would be important to add more granularity to this field (maybe rename it to radiusInKilometers/radiusInMiles, or add a model with a unit type and the radius value), so then the different atlas adapters will translate the model.

Reply:
Got it.. I thought you were mentioned the data type